modules/react/combobox/lib/ComboboxInput.tsx

```diff
@@ -56,11 +56,8 @@ export const useComboboxInput = composeHooks(
           dispatchInputEvent(event.currentTarget as HTMLElement, '');
         }
         if (event.key === 'Enter' && !event.metaKey && model.state.visibility === 'visible') {
-          if (
-            document
-              .querySelector(`[data-id="${model.state.cursorId}"]`)
-              ?.getAttribute('aria-disabled') !== 'true'
```

Comment:
was this chaining?

Reply:
Originally the conditional was:

```ts
document.querySelector(`[data-id="${model.state.cursorId}"]`)
```

"If the element exists, continue". Later we added the requirement that the element should not be disabled via `aria-disabled`. The updated logic turned out to be "If the element exists AND is not disabled OR the element does not exist" when it should be "If the element exists AND is not diabled"